config-windows.md

```diff
@@ -94,3 +94,42 @@ The following parameters can be specified:
         }
    }
 ```
+
+## <a name="configWindowsCredentialSpec" />Credential Spec
+
+You can configure a container's group Managed Service Accounts (gMSAs) via the OPTIONAL `credentialspec` field of the Windows configuration. For more information about gMSAs, see [Active Directory Service Accounts for Windows Containers][gMSAOverview]. For more information about tooling to generate a gMSA, see [Deployment Overview][gMSATooling]. 
+
+
+### Example
+
+```json
+    "windows": {
+        "credentialspec": {
```

Comment:
This value is a JSON object.  Your current docs don't have much to say on the value type (using the `credentialspec` (object, OPTIONAL) …` pattern would help with this), but your current Go type has it listed as a string (which will break if you try to read in your example here).